tests/browser/test/test_setup.js

```diff
@@ -116,23 +116,14 @@ async function getBlockElementById(browser, id) {
 /**
  * @param browser The active WebdriverIO Browser object.
  * @param categoryName The name of the toolbox category to find.
- * @return A Promise that resolves to the root element of the toolbox
- *     category with the given name, as an interactable browser element.
+ * @return The root element of the toolbox category with the given name, as an interactable browser element.
```

Comment:
Why was the previous doc wrong? As always, I'm shaky on Promises, but I thought that the return value of an async function is always a promise, and the only type question is what it resolves to.

If that's wrong, I also need to update the other functions in this file.

Reply:
No, you're right, it does. I think I changed it because I added the await before returning so I was thinking the promise would be resolved before returning, but because it's async it just gets wrapped in another promise. @_@

Reply:
Cool--that matches my understanding.